packages/prediction-app/src/components/NewEvaluationForm/Sections/DatasetConfiguration/DataMappingModal/DataMappingModal.tsx

```diff
@@ -58,6 +60,8 @@ export const DataMappingModal = ({
                             <FeatureMappingItem
                                 feature={model.target}
                                 onMapping={handleTargetMapping}
+                                existingMapping={localState.targetMapping?.dataItem}
+                                onResetField={() => resetCovariateMapping(model.target.name || '')}
```

Comment:
NIT: but maybe cleaner to conditionally call the function, instead of calling with empty string?